data/i18n/updateTranslations.sh

```diff
@@ -40,7 +40,8 @@ $CMD ../../data/i18n/hydrogen_el.ts
 $CMD ../../data/i18n/hydrogen_gl.ts
 $CMD ../../data/i18n/hydrogen_sr.ts
 $CMD ../../data/i18n/hydrogen_ca.ts
-$CMD ../../data/i18n/hydrogen_uk.ts
+$CMD ../../data/i18n/hydrogen_en.ts
+$CMD ../../data/i18n/hydrogen_en_GB.ts
```

Comment:
It seems you removed the ukrainian translation update by mistake.

Reply:
Well spotted, thanks!